### PR TITLE
Label Feedback Responses

### PR DESCRIPTION
**Pull request by thelenw**
* Labled empty feedback responses
* UI adjustments

### CODE REVIEW

File: web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.css [hunk 1]
```diff
@@ -24,6 +24,17 @@
...
   padding: 10px;
+  display: flex;
+  align-items: center;
+}
+
+.caution-icon {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  background-image: url('path-to-caution-icon.png'); /* Add the correct icon path */
```
Comment (mkimberlin) [inline web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.css:35]: This looks suspect...

File: web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx [hunk 2]
```diff
@@ -85,7 +86,21 @@ const ViewFeedbackResponses = () => {
           ? requests
           : [requests]
         : [];
-      return await getQuestionsAndAnswers(requests, cookie);
+      const res = await getQuestionsAndAnswers(requests, cookie);
+
+      if (res) {
+        console.log('Retrieved Questions and Answers:', res); // Debugging log
```
Comment (mkimberlin) [inline web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx:92]: as a rule, we try not to leave console.log calls in the codebase.

File: web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx [hunk 3]
```diff
@@ -278,6 +284,41 @@ const ViewFeedbackResponses = () => {
...
+              >
+                Section 2: You can provide more detailed context to your review of this team member below. Be as specific as possible!
+              </Box>
+            );
+          }
+
+          // Handle other questions by removing "Q#: "
+          const questionText = question.question.replace(/^Q\d+: /, '');
+          const hasResponses = question.answers.length > 0;
+
```
Comment (mkimberlin) [inline web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx:321]: The above changes are problematic. They assume that the template used for reviews is static, but that is not the case.  Doing this creates an even tighter coupling to the data (template questions) than we already have. I would roll this bit back and stick to just addressing the items mentioned in the issue.

Comment (thelenw) [reply web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx:309]: Done.

File: web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx (hunk 2 above)
Comment (thelenw) [reply web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx]: Existing console.log calls in pertinent files have been removed.

File: web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.css (hunk 1 above)
Comment (thelenw) [reply web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.css]: Fixed.

**Comment by thelenw** [comment]
* Implemented logic to display "No input" for questions with ```inputType: 'NONE'``` in ```components/view_feedback_responses/```

* For empty answers, system now displays "⚠️ No response submitted"

* Adjusted the filtering logic for search and responders to only display relevant answers while handling fallback cases properly.

* Improved logic for setting answers in ```FeedbackResponseCard``` to cover edge cases and non-string values.

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**